docs/assignment1.md

```diff
@@ -1,62 +1,80 @@
+| F2  | Movements  | The player may move freely between different areas in four directions as described by the `move` command above, or by specifying a unique area name (optionally given in the description of the current area). Some areas may be blocked by obstacles and therefore inaccessible. Obstacles may be removed by using a specific item in their presence (e.g. "use key" might open a door).  |
+| F3  | Areas | All areas will be represented as nodes in a graph, where edges between nodes indicate two areas are reachable from one another. Each area will have a description so the player can distinguish between different type of areas, however, some areas may not be unique.  |
+| F4  | Obstacles | An obstacle is a special type of object which prohibits movement between two specific areas until a certain item is used in the same area that they inhabit. Having obstacles be generic like this allows some more complex behaviour to exhibit itself - for example, defeating a monster may have them drop an item that "unlocks" an obstacle, therefore inhibiting progress until the monster is defeated. |
+| F5  | Items | The player may acquire and use items throughout the game. Items will be stored in an inventory of infinite size, which may be accessed using the `inventory` command (i.e. `inventory` will print the names of the currently held items). Some items are consumed upon first use, whereas other items persist throughout the game. Several commands may be used in combination with an item: </br> - `interact` acquires the item and places it in the player's inventory. </br> - `look` briefly describes the item. </br> - `use` queries the item, asking it what it may be used for, and if appropriate for the situation, uses it (e.g. `use key` might consume the key if in the presence of the appropriate door, otherwise the inventory will remain unchanged).  |
+| F6  | NPCs | Throughout the game, several NPCs will appear which can be interacted with using the `interact` command. NPCs may be friendly or hostile and could hold valuable or important items for the user to acquire.  |
+| F7  | Stats | The player has two main stats, `health` and `strength`, which they will use for combat. Naturally, `health` is an indicator of the player's remaining vitality (reaching 0 health will result in losing the game), whereas `strength` determines whether or not the player can defeat an enemy in combat, and how much damage they take during the battle.  |
+| F8  | Combat | Whenever a player engages in combat, their strength is compared to that of the enemy combatant. The outcome of the fight is determined by the difference in strength between both the player and their enemy:</br> - player strength < enemy strength: enemy survives, player loses a significant chunk of health </br> - player strength >= enemy strength: enemy defeated, player loses some health</br> - player strength >> enemy strength: enemy defeated, player loses no health |
```

Comment:
Please provide more information about how the combat phase is structured and how it fits with the rest of the gameplay. Is the combat initiated by the interact command, does it have only one phase and what happens to the player and npc stats afterwards. Define precisely what is meant by player strenght >= or >>